src/lib/components/composites/paper-components/paper-view/cards/PaperResearchContextCard.svelte

```diff
@@ -33,13 +38,15 @@
     }: PaperResearchContextCardProps = $props();
 
     const reviewInfoTab = { value: "1", label: "Review Information" };
-    const referencesTab = { value: "2", label: "Forward/Backward References" };
+    const forwardReferencesTab = { value: "2", label: "Citations" };
```

Comment:
```suggestion
    const forwardReferencesTab = { value: "2", label: "Backward References" };
```

For consistency? Same for the name of the list?

Reply:
Or is there a reason, why you choose this wording?

Reply:
No, I thought this was clearer, but as discussed, we keep it like that.